Users/Tutorial.rst

```diff
@@ -148,7 +148,7 @@ This should yield something like this:
 
 ::
 
-    [Default]
+    [cli]
```

Comment:
That shouldn't be encouraged:
```
[WARNING][14:51:40] 'cli' is an internally reserved section name. It may have been generated into your coafile while running coala with `--save`. The settings in that section will inherit implicitly to all sections as defaults just like CLI args do. Please change the name of that section in your coafile to avoid any unexpected behavior.
```
Use maybe `all`, as this name is used as an example in the log help texts.

Reply:
I have changed and used `all` in the tutorial below. IMHO we should keep `cli` here because this is just the output of a command. :smiley:

Reply:
Ah it's generated by `--save`? Then it's fine 👍